Python/instrumentation.c

```diff
@@ -276,6 +276,12 @@ _PyInstruction_GetLength(PyCodeObject *code, int offset)
     }
     assert(opcode != 0);
     assert(!is_instrumented(opcode));
+    if (opcode == ENTER_EXECUTOR) {
+        _PyExecutorObject *exec = code->co_executors->executors[_PyCode_CODE(code)[offset].op.arg];
```

Comment:
I suggest to write the code on two line to make it easier to read:

```suggestion
        int exec_index = _PyCode_CODE(code)[offset].op.arg;
        _PyExecutorObject *exec = code->co_executors->executors[exec_index];
```